resources/views/world/_feature_entry.blade.php

```diff
@@ -16,14 +16,14 @@
         <div class="world-entry-text parsed-text">
             {!! $feature->parsed_description !!}
         </div>
-        @if($feature->altTypes->count())
+        @if($feature->altTypesVisible->count())
```

Comment:
So a fun trick you can do to be able to use scopes on relations is append `()` to the relation to access it more query-style than collection-style. But then you need to also have `get()` at the end or it doesn't actually _return anything_.
So this becomes `$feature->altTypes()->visible()->get()->count()` iirc (should double-check, I just woke up). Which is "messier", but not actually considering this is the only place in which it matters.

Reply:
Edited, tested- worked!